frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -18,12 +18,17 @@
               @blur="$v.organization.$touch"
               @change="$v.organization.$touch"
             />
+
             <lf-field-messages
               :validation="$v.organization"
               :error-messages="{
                 required: 'This field is required',
               }"
             />
+            <lf-field-message v-if="!isEdit && hasSameOrganization" type="warning" class="!mt-2">
```

Comment:
Could you remove !isEdit?
This should be applied when a user is adding a work experience or editing it